docs/_templates/layout.html

```diff
@@ -68,12 +68,19 @@ <h1 class="logo">
         <ul class="menu">
           <li>
             <a href="
-https://googlecloudplatform.github.io/gcloud-node" title="Node.js docs page">
+https://googlecloudplatform.github.io/gcloud-node/#/docs" title="Node.js docs page">
```

Comment:
Just making sure we're rebased and all that -- since this commit technically does two things, right? Adds Ruby and updates Node.